pages/api/profiles.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from '@prisma/client'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+const prisma = new PrismaClient()
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  if (req.method !== 'GET') {
```

Comment:
**nitpick:** You will need to add here more methods later - what about creating some reusable method for handling `req.method` like: 

```javascript
handleRequest(req.method, callbackFn) {
    case "GET":
    return callbackFn();
    case "POST":
    return callbackFn();
    ...
}
```

Reply:
Should this function take req, res and return res, or should it only contain logic?

Reply:
I think it can take both req, and res, so it's more useful